README.md

```diff
@@ -0,0 +1,49 @@
+## Merging selinux-policy-contrib repository with selinux-policy
+
+Currently, SELinux policy packages in Fedora use 2 repositories:
+base [1] and contrib [2].
+This division into two repos is merely a historical artifact and
+the main effect is make dealing with the repos more difficult.
+
+We work on a change to merge these repos into one, containing sources from both.
+All the changes would affect both the repos, branches rawhide, and future branches
+f34 and newer.
+After the merge, when working in the rawhide branch, only the base repo would
+be used; the corresponding contrib branch would be archived and not used
+any longer. The contrib repo's commit history would be a part of the base repo.
+Stable branches (f33, f32, all older ones) would remain unchanged.
+The plan is to make it happen in the week starting with Monday, November 23rd.
+
+It mainly is an internal change of where the git repository is stored and
+how it is referenced. After the merge, there will be these notable changes
+inside the repo:
+- all files previously accessible from the root directory in the selinux-policy-contrib repo will be in selinux-policy base repo, directory `policy/modules/contrib/`
+- no change for work in selinux-policy base repo
+
+### How users will be affected?
+There is no change for users.
+
+### How custom selinux-policy developers will be affected?
+No change for policy writing 
+other then where to look for for the previous contrib modules.
+
+Scripts, data, specfile, etc. in the dist git will be updated to use the new location.
+
+### How selinux-policy contributors will be affected?
+No change other then where to look for for the previous contrib modules and where to submit pull requests.
+
+### How to submit pull requests?
```

Comment:
"Where to" sounds more appropriate here (and below).

Reply:
Updated.